Tusky 13.0 beta 1 crashes when opening specific audio file

When I open the audio file in the toot https://chaos.social/@leah/105288490769076928, Tusky crashes.

Let me know if I can help out with any logs and how I can gather them if needed. 

* * * *
- Tusky Version: 13.0 beta 1
- Android Version: 10
- Android Device: Motorola Moto G8+
- Mastodon instance (if applicable): chaos.social

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

Thx for reporting, I saw this in the reports and was wondering how to reproduce it

Here is a stacktrace
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.keylesspalace.tusky.test/com.keylesspalace.tusky.ViewMediaActivity}: java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter previewUrl
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3270)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
     Caused by: java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter previewUrl
        at com.keylesspalace.tusky.entity.Attachment.<init>(Unknown Source:12)
        at com.keylesspalace.tusky.entity.Attachment$Creator.createFromParcel(Unknown Source:58)
        at com.keylesspalace.tusky.entity.Attachment$Creator.createFromParcel(Unknown Source:28)
        at com.keylesspalace.tusky.viewdata.AttachmentViewData$Creator.createFromParcel(Unknown Source:9)
        at com.keylesspalace.tusky.viewdata.AttachmentViewData$Creator.createFromParcel(Unknown Source:9)
        at android.os.Parcel.readParcelable(Parcel.java:2973)
        at android.os.Parcel.readValue(Parcel.java:2866)
        at android.os.Parcel.readListInternal(Parcel.java:3307)
        at android.os.Parcel.readArrayList(Parcel.java:2450)
        at android.os.Parcel.readValue(Parcel.java:2887)
        at android.os.Parcel.readArrayMapInternal(Parcel.java:3244)
        at android.os.BaseBundle.initializeFromParcelLocked(BaseBundle.java:292)
        at android.os.BaseBundle.unparcel(BaseBundle.java:236)
        at android.os.Bundle.getParcelableArrayList(Bundle.java:1005)
        at android.content.Intent.getParcelableArrayListExtra(Intent.java:7957)
        at com.keylesspalace.tusky.ViewMediaActivity.onCreate(ViewMediaActivity.kt:110)
        at android.app.Activity.performCreate(Activity.java:7802)
        at android.app.Activity.performCreate(Activity.java:7791)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1299)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3245)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409) 
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016) 
        at android.os.Handler.dispatchMessage(Handler.java:107) 
        at android.os.Looper.loop(Looper.java:214) 
        at android.app.ActivityThread.main(ActivityThread.java:7356) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

--- a/app/src/main/java/com/keylesspalace/tusky/entity/Attachment.kt
+++ b/app/src/main/java/com/keylesspalace/tusky/entity/Attachment.kt
@@ -28,7 +28,7 @@ import kotlinx.android.parcel.Parcelize
 data class Attachment(
         val id: String,
         val url: String,
-        @SerializedName("preview_url") val previewUrl: String,
+        @SerializedName("preview_url") val previewUrl: String?, // can be null for e.g. audio attachments
         val meta: MetaData?,
         val type: Type,
         val description: String?,

